src/components/subheader/DescriptionText.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import address from './constants/address'
+
+function DescriptionText(){
+
+    const {name, type, city } = address;
+    
+    return (
+        <>
+            <p className="description-heading">{name}</p>
```

Comment:
Start using props system else your components are kinda static because if tomorrow data is coming from some API, you'll obviously pass it down the tree, you'll not have access to it everywhere and your code will break.